install/kubernetes/ingress-sni-zvpn.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: envoy-config
+  namespace: istio-system
+data:
+  envoy.yaml: |
+    stats_config:
+      use_all_default_tags: false
+
+    admin:
+      access_log_path: /dev/stdout
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 15000 }
+
+    dynamic_resources:
+      lds_config:
+        api_config_source:
+          api_type: REST_LEGACY
+          refresh_delay: 10s
```

Comment:
This sucks.. coz we have global mesh configs to control this. Is there any way we can push this into the pilot agent to spit out the YAML based on global configs?

Reply:
and why do you need LDS at all for ingress? You just need CDS, RDS and EDS

Reply:
LDS is need for one of the zero-vpn solutions - so we can add ports on the ingress for mesh expansion.

Reply:
Also for TCP ingress, when Gateway is added.

Reply:
Please maintain compatibility with the mesh config. I don't see how to turn on/off tracing, for example, using this YAML.

Reply:
That’s a good catch kuat. Costin this config is going to disable tracing at Ingress effectively. One option is to always enable tracing and bear with Envoy complaining that there is no associated cluster.

Reply:
Well, I'll match whatever the current agent does - just didn't get time to test the v2 way of configuring that, and I'm trying to get the template working first... WIP...